mantidproject/mantid
# Loader for D1B instrument

**Description of work.**
This PR adds a loader for ILL's instrument D1B.
Since the 2theta offset is not yet included in the nexus files, a field is added so the user can provide it himself.



**To test:**
- Load `External/Testing/Data/UnitTest/ILL/D1B/473432.nxs`.
- Check there is data, and that the instrument looks fine.


Fixes #29848 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
